plugins/bgp/GlobalConfigurator/api_globalConfigurator.go

```diff
@@ -0,0 +1,38 @@
+package GlobalConfigurator
+
+import (
+	"github.com/contiv/bgp-vpp/plugins/bgp/model"
+	"github.com/ligato/cn-infra/db/keyval"
+	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/cn-infra/logging/logrus"
+)
```

Comment:
we should not need both loggin and logrus, logging should be sufficient (in other words, all function that you should be using are defined in github.com/ligato/cn-infra/logging)